src/Network/HTTP/StackClient.hs

```diff
@@ -41,7 +41,7 @@ module Network.HTTP.StackClient
   , partFileRequestBody
   , partBS
   , partLBS
-  , setGithubHeaders
+  , setGitHubHeaders
```

Comment:
I'm not a Haskell dev. But, I believe this is just a forward reference to the thing below, so it seems harmless